common/inc/stdc++/linux/exception

```diff
@@ -33,11 +33,11 @@ namespace std
 	class exception
 	{
 	public:
-		exception() throw();
```

Comment:
AFAIK C++11 deprecated throw in favor of the noexcept specifier. Shouldn't "throw()" be replaced with "noexcept" instead of just removing "throw()"?